application/cms/views.py

```diff
@@ -1074,6 +1059,43 @@ def new_version(topic, subtopic, measure, version):
     )
 
 
+@cms_blueprint.route("/<topic>/<subtopic>/<measure>/<version>/copy", methods=["POST"])
+@login_required
+@user_can(COPY_MEASURE)
+def copy_measure_page(topic, subtopic, measure, version):
+    topic_page, subtopic_page, measure_page = page_service.get_measure_page_hierarchy(topic, subtopic, measure, version)
+
+    # These may be changed by the call to create_copy so remember them here, in case we need them for error handling
+    original_measure_guid = measure_page.guid
+    original_measure_version = measure_page.version
+
+    try:
+        copied_page = page_service.create_copy(measure, version, version_type="copy", created_by=current_user.email)
+        return redirect(
+            url_for(
+                "cms.edit_measure_page",
+                topic=topic_page.guid,
+                subtopic=subtopic_page.guid,
+                measure=copied_page.guid,
+                version=copied_page.version,
+            )
+        )
+    except Exception as e:
+        # Any errors here are unexpected so can't be more specific about what we catch
```

Comment:
I'm still unconvinced by the need to explicitly catch all generic unexpected Exceptions here. Feels to me like we should do the same as for other database actions - ie let the exception get reported via our logging and display a 500 error to the user?

Reply:
Yeah that seems reasonable.  If there's nothing we expect to handle then throwing the error should be fine.